civiwiki/api/models/article.py

```diff
@@ -2,6 +2,15 @@
 from django.db import models
 
 # Create your models here.
+class ArticleManager(models.Manager):
+   def serialize(self, account, filter=None):
+      data ={
+         "category": Topic.category,
+         "topic": Topic.topic,
+         "bill": Topic.bill,
+         "all_objects": Topic.objects,
```

Comment:
so you actually don't need to print out the objects attribute on any class, **objects** is actually the definition of the **manager** that you created. Its for our own internal use to be able to call functions like. **Account.objects.filter()** & **Account.objects.serialize()**. Should've told you this, apologies!